.github/workflows/weekly-release.yml

```diff
@@ -0,0 +1,41 @@
+name: Weekly Release
+
+on:
+  schedule:
+    - cron: '0 10 * * 3'  # every wednesday at 10 am
```

Comment:
I'd prefer to release earlier. Given the max review time, a release might get published only on 10am Fridays with another delay until user base will install it. I think Monday would be the best time to release, imo.

Also what'll be the timezone here?

Reply:
It's in UTC, I'll change to Mondays :).